age/tui.go

```diff
@@ -1,5 +1,6 @@
 // These functions have been copied from the age project
 // https://github.com/FiloSottile/age/blob/v1.0.0/cmd/age/encrypted_keys.go
+// https://github.com/FiloSottile/age/blob/266c0940916da6bfa310fdd23156fbd70f9d90a0/tui/tui.go
```

Comment:
Commit `266c0940916da6bfa310fdd23156fbd70f9d90a0` is not part of FiloSottile/age. Can you use https://github.com/FiloSottile/age/blob/3d91014ea095e8d70f7c6c4833f89b53a96e0832/cmd/age/tui.go instead (and also replace the code with the code from that commit)? It is also preferable to use lower-case identifiers (as in the original tui.go) so that the symbols defined in here are private to SOPS' `age` package.

Reply:
Done